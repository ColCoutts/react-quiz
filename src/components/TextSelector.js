import React from 'react';
import PropTypes from 'prop-types';

function TextSelector({ text }) {
  return (
    <div>
      <input name="text" value={text} onChange={this.handleChange}></input>
    </div>
  )
}

TextSelector.propTypes = {
  text: PropTypes.string.isRequired
};

export default TextSelector
