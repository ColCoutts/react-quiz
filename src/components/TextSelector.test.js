import React from 'react';
import { shallow } from 'enzyme';
import TextSelector from './TextSelector';

describe('textSelector component', () => {
  it('renders a snapshot', () => {
    const text = 'hello';
    const wrapper = shallow(<TextSelector text={text} />);
    expect(wrapper).toMatchSnapshot();
  });
});
