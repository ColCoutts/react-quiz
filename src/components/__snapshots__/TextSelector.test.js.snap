// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textSelector component renders a snapshot 1`] = `
<Fragment>
  <input
    name="text"
    value="hello"
  />
</Fragment>
`;
